hal/src/xenon/pinmap_hal.c

```diff
@@ -64,60 +64,6 @@ const uint8_t NRF_PIN_LOOKUP_TABLE[48] = {
 
 #endif // PLATFORM_ID == PLATFORM_XENON
```

Comment:
We can probably remove this macro condition. Same source file with regard the Argon platform.